images/Dockerfile.ubuntu

```diff
@@ -87,6 +89,12 @@ RUN ln -s /usr/bin/grub-editenv /usr/bin/grub2-editenv
 RUN systemctl enable systemd-networkd
 RUN systemctl enable ssh
 
+# workaround https://github.com/systemd/systemd/issues/12231
```

Comment:
[The fix](https://github.com/systemd/systemd/pull/12281) is in v242 of systemd (according to github). `ubuntu:rolling` has systemd v252:

```
root@3935f78807f2:/# systemd --version
systemd 252 (252.5-2ubuntu3)
```

Are we sure we still need this patch?

Reply:
yeah, it was failing with an overlayfs issue if the patch was missing :(